packages/tabs/examples/with-non-tabs-components-interweaved.example.js

```diff
@@ -0,0 +1,52 @@
+import "@reach/tabs/styles.css";
+
+import React from "react";
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@reach/tabs";
+
+export const name = "With Non-Tabs Components Interweaved";
+
+const tabsStyle = {
+  width: 400,
+  boxShadow: "1px 1px 5px hsla(0, 0%, 0%, 0.25)"
+};
+
+const tabListWrapperStyle = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "space-between",
+  border: "1px solid darkslategray",
+  padding: "4px 0"
+};
+
+export const Example = () => (
```

Comment:
Not sure if this example is the best display of showing the decoupling this PR affords. I'm happy to take suggestions on better examples if people have them.

Here's a gif of the example above:

![tabs-with-context](https://user-images.githubusercontent.com/3409645/70598718-4a179c00-1ba0-11ea-988e-28593820e5fa.gif)